host_guest/README.md

```diff
@@ -36,6 +37,6 @@ All the host-guest system files in the `SAMPLing/` directory were prepared using
 - 5 docked complexes were generated with OpenEye `2017.6.1`.
 - Hosts and guests were both parametrized with GAFF v1.8 and antechamber. AM1-BCC charges were generated using OpenEye's QUACPAC toolkit through `openmoltools 0.8.1`.
 - The systems were solvated in a 12A buffer of TIP3P water molecules using tleap. ParmEd `2.7.3` was used to remove some of the water molecules from the OA complexes to reduce them to have the same number of waters.
-- The systems' net charge was neutralized with Na+ and Cl- ions. More ions were added to reach the ionic strength of 10mM for OA/TEMOA systems and 25mM for CB8.
+- The systems' net charge was neutralized with Na+ and Cl- ions. More Na+ and Cl- ions were added to reach the ionic strength of 60mM for OA/TEMOA systems and 150mM for CB8 to simulate the effect of the 10mM and 25mM sodium phosphate buffer used in their respective experiments.
```

Comment:
@andrrizzi - just confirming; did you actually change the files for the SAMPLing challenge? Or did you forget to update this before? Or is this an update where you are putting in the conditions you're using for the reference calculations into the SAMPLing section on accident?

Reply:
The SAMPLing challenge files are unchanged. I just fixed the preparation description that mixed up the buffer ionic strength with its concentration.